app/Actions/Dislike/DislikeReviewAction.php

```diff
@@ -44,16 +45,31 @@ public function execute(DislikeReviewRequest $request): DislikeReviewResponse
         )->first();
         
         if ($like) {
+            broadcast(new AttitudeSetEvent(
```

Comment:
I think it would be a better design if you used called event here and in the listener did your action with a broadcasting.